tests/integration/test_patch.py

```diff
@@ -420,3 +420,35 @@ def on_click(_):
 
     dash_duo_mp.wait_for_element("#click2").click()
     dash_duo_mp.wait_for_text_to_equal("#output", "click 2")
+
+
+def test_pch005_clientside_duplicate(dash_duo):
+    app = Dash(__name__)
+
+    app.layout = html.Div(
+        [
+            html.Button("Click 1", id="click1"),
+            html.Button("Click 2", id="click2"),
+            html.Div("initial", id="output"),
+        ]
+    )
+
+    app.clientside_callback(
+        "function onClickOne() { return 'click1';}",
+        Output("output", "children", allow_duplicate=True),
+        Input("click1", "n_clicks"),
+        prevent_initial_call=True,
+    )
+    app.clientside_callback(
+        "function onClickTwo(){ return 'click2';}",
```

Comment:
Would a more comprehensive test be to have an anonymous function? To test the `uuid.uuid4.hex()` thing as above. So like `() => { return 'click2' }`?

Reply:
Not really, the function name is actually discarded/unimportant when the script inline, I had fixed on the rendered first but the test still failed so I tried a few thing like adding name on the function to found out where it was getting the name and replaced with the hex.